src/app/components/game-center/game-center.component.spec.ts

```diff
@@ -20,4 +38,8 @@ describe('GameCenterComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  // it('should check', () => {
+  //   expect(component).toBeTruthy();
+  // });
```

Comment:
```suggestion
   it('should check', () => {
     expect(component).toBeTruthy();
   });
```

Do not upload commented code